docs/querying/examples.md

```diff
@@ -39,7 +39,7 @@ To select all HTTP status codes except 4xx ones, you could run:
 
 ## Subquery
 
-Return the 5-minute rate of the `http_requests_total` metric for the past 30 minutes, with a resolution of 1 minute.
+Return the 5-minute [rate](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) of the `http_requests_total` metric for the past 30 minutes, with a resolution of 1 minute.
```

Comment:
```suggestion
Return the 5-minute [rate](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) of the `http_requests_total` metric for the past 30 minutes, with a resolution of 1 minute.
```

We should use a relative link instead of pointing to "prometheus.io"